src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,22 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
That's not critical cause the main purpose of this task is cascades but it's not recommended to use Eager fetch type, instead use join fetch with lazy fetchType

Reply:
ok